documentation/library_guide/pstl/dpcpp_policies_usage.rst

```diff
@@ -175,11 +175,22 @@ allow you to pass SYCL buffers to parallel algorithms. These functions accept
 a SYCL buffer and return an object of an unspecified type that satisfies the following
 requirements:
```

Comment:
I think, I understand what we are trying to say. In that case I would not use the word _requirements_ at all because those are not our requirements (to let's say function argument). Those are our guarantees. In other words, that is the API we guarantee that object has and it's available for calling on the user side. Basing on that, please see my suggestions.

Reply:
Actually I don't see differences. Here we say that users can rely on following functionality. Isn't it requirements?

Reply:
But I do. When you think about the requirements you basically asking something to fulfill those requirements but if you are promising something you basically provide the guarantees. That's how I see it.